vmss-prototype/vmss-prototype

```diff
@@ -899,23 +899,30 @@ def in_cluster(sub_args, func):
              '--name', cloud
              ])
 
-        login_cmd = ['az', 'login',
-                    '--username', username,
-                    '--password', password,
-                    '--tenant', tenant,
-                    '--service-principal'
-                    ]
+        if username == "msi":
+            login_cmd = ['az', 'login', '--identity']
+        else:
+            login_cmd = ['az', 'login',
+                        '--username', username,
+                        '--password', password,
+                        '--tenant', tenant,
+                        '--service-principal'
+                        ]
```

Comment:
Nice - if this is that simple - have you tested this?

Reply:
tested `az login --identity` on an MSI-enabled host, yeah

Reply:
:-)  Thanks - I was just worried about the lines above reading elements that are not technically needed for MSI.

Anyway, thanks for doing this!